packages/blockchain/src/blockchain.ts

```diff
@@ -581,8 +581,8 @@ export class Blockchain implements BlockchainInterface {
     }
 
     if (header.common.isActivatedEIP(7685)) {
-      if (header.requestsRoot === undefined) {
-        throw new Error(`requestsRoot must be provided when EIP-7685 is active`)
+      if (header.requestsHash === undefined) {
+        throw new Error(`requestsHash must be provided when EIP-7685 is active`)
```

Comment:
I don't think this check needs to be there, isn't this already validated in the constructor (so in the `BlockHeader` constructor)? I also don't see similar checks for other non-standard properties here?

Reply:
infact we do have such checks, see just above this for 4844